src/details/ArborX_DetailsTreeNodeLabeling.hpp

```diff
@@ -0,0 +1,116 @@
+/****************************************************************************
+ * Copyright (c) 2017-2021 by the ArborX authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the ArborX library. ArborX is                       *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#ifndef ARBORX_DETAILS_TREE_NODE_LABELING_HPP
+#define ARBORX_DETAILS_TREE_NODE_LABELING_HPP
+
+#include <ArborX_DetailsHappyTreeFriends.hpp>
+#include <ArborX_Exception.hpp>
+
+#include <Kokkos_Core.hpp>
+
+namespace ArborX
+{
+
+namespace Experimental
```

Comment:
@dalg24 I thought we decided this should be in `Details` and not `Experimental`?

Reply:
Oversight

Reply:
Fixed